draft-ietf-jsonpath-base.md

```diff
@@ -298,19 +298,19 @@ $.store.book[?(@.price < 10)].title
 
 {{tbl-overview}} provides a quick overview of the JSONPath syntax elements.
 
-| JSONPath       | Description           |
-|----------------|-----------------------|
-| `$`                | the root node                                                                |
-| `@`                | the current node                                                             |
-| `.` or `[]`        | child operator                                                               |
-| n/a                | parent operator                                                              |
-| `..`               | nested descendants                                                           |
-| `*`                | wildcard: all member values/array elements regardless of their names/indices |
-| `[]`               | subscript operator: index current node as an array (from 0)                  |
-| `[,]`              | list operator: allows combining member names, array indices, and slices |
-| `[start:end:step]` | array slice operator                                                         |
-| `?()`              | applies a filter expression                                                  |
-| `()`               | expression, e.g., for indexing                                               |
+| JSONPath    | Description                                                                                                                         |
+|-------------|-------------------------------------------------------------------------------------------------------------------------------------|
+| `$`         | [the root node](#root-selector)                                                                                                     |
+| `@`         | the current node (within [filter selectors](#filter-selector))                                                                      |
+| `.name`     | child selectors for JSON objects: [dot selector](#dot-selector)                                                                     |
+| `['name']`  | child selectors for JSON objects: [index selector](#index-selector)                                                                 |
+| `..`        | nested descendants: [descendant selector](#descendant-selector)                                                                     |
+| `*`         | [dot wildcard selector](#wildcard), [index wildcard selector](#index-wildcard-selector): all child member values and array elements |
```

Comment:
I'd prefer a consistent format for descriptions. Some, such as this one, have the text and the link in the opposite order to the other descriptions. This may not work everywhere, but I think the rendered version looks nicer:
```suggestion
| `*`         | all child member values and array elements: [dot wildcard selector](#wildcard), [index wildcard selector](#index-wildcard-selector) |
```